packages/design-system-docs/src/scripts/components/__tests__/ReactPropDoc.test.js

```diff
@@ -40,13 +45,23 @@ describe('ReactPropDoc', () => {
   it('should render type', () => {
     const data = shallowRender();
 
-    expect(data.wrapper.find('td').at(1).text()).toBe(data.props.type.name);
+    expect(
```

Comment:
Where these changes from the linter? What happens if you run `yarn lint`?

Reply:
Scott - thanks for testing. I've removed the `multiple` props from the example file. To test this, modify `ChoiceList.example.jsx` to replace line 31 `type="checkbox"` with `multiple`.

Bernard - these are my changes, I'll need to go and resolve the conflicts. I've changed `toBe` with `toMatch`.